helpers/constants.js

```diff
@@ -93,7 +94,7 @@ var constants = {
 	},
 	signatureLength: 196,
 	// WARNING: When changing totalAmount you also need to change getBlockRewards(int) SQL function!
-	totalAmount: 10000000000000000,
+	totalAmount: new bignum(10000000000000000),
```

Comment:
Please check the [failing build](https://jenkins.lisk.io/job/lisk-core/view/change-requests/job/PR-847/) on Jenkins. There are still places in the code where we use `constants.totalAmount `expecting it to be a number, not a bignum.